scripts/verify_conda_libmesh.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# * This file is part of the MOOSE framework
+# * https://www.mooseframework.org
+# *
+# * All rights reserved, see COPYRIGHT for full restrictions
+# * https://github.com/idaholab/moose/blob/master/COPYRIGHT
+# *
+# * Licensed under LGPL 2.1, please see LICENSE for details
+# * https://www.gnu.org/licenses/lgpl-2.1.html
+
+import os
+import sys
+try:
+    from conda_build.metadata import MetaData
+except ImportError:
+    print('conda_build module not available.')
+    sys.exit(1)
+
+
+def main():
+    """Check to see if the user's current conda environment is up to date."""
+
+    # Locate MOOSE directory
+    os.chdir(os.path.abspath(os.path.dirname(__file__)))
+    moose_dir = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..'))
+    if not os.path.exists(moose_dir):
+        print('Failed to locate MOOSE, specify the MOOSE_DIR environment variable.')
+        sys.exit(1)
+    os.environ['MOOSE_DIR'] = moose_dir
+
+    libmesh_recipe = os.path.join(moose_dir, "conda", "libmesh", "meta.yaml")
+    if os.path.exists(libmesh_recipe):
+        meta = MetaData(libmesh_recipe)
+        libmesh_version = '-'.join([meta.meta['package']['name'],
+                                    meta.meta['package']['version'],
+                                    meta.meta['build'].get('string', '*')]) + '.json'
+
+    conda_prefix = os.getenv('CONDA_PREFIX')
+    if conda_prefix is None:
+        print("Failed to locate CONDA_PREFIX, ensure your environment is activated.")
+        sys.exit(1)
+
+    conda_libmesh = os.path.join(conda_prefix, "conda-meta", libmesh_version)
+    if not os.path.exists(conda_libmesh):
+        print("Your version of moose-libmesh is not current.")
+        print("Please run conda update --all.")
```

Comment:
this might not help. this is where we need to run the conda search commandline I posted in the slack channel and determine if the required version even exists yet!